package.json

```diff
@@ -17,7 +17,7 @@
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
-    "chromedriver": "^78.0.0",
+    "chromedriver": "^79.0.0",
```

Comment:
As per [the list of software installed on the Azure Pipelines agent](https://github.com/actions/virtual-environments/blob/a8415e472607ec99d8715dbfcd8c83a10d25e4b3/images/linux/Ubuntu1804-README.md), this should now be chromedriver ^80.0.0.